kurtosis-devnet/cmd/main.go

```diff
@@ -124,10 +124,74 @@ func localContractArtifactsOption(cfg *config, server *staticServer) tmpl.Templa
 	})
 }
 
+func localPrestateOption(cfg *config, server *staticServer) tmpl.TemplateContextOptions {
+	prestateBuilder := build.NewPrestateBuilder(
+		build.WithPrestateBaseDir(cfg.baseDir),
+		build.WithPrestateDryRun(cfg.dryRun),
+	)
+
+	return tmpl.WithFunction("localPrestate", func() (string, error) {
+		// Create build directory with the final path structure
+		buildDir := filepath.Join(server.dir, "proofs", "op-program", "cannon")
+		if err := os.MkdirAll(buildDir, 0755); err != nil {
+			return "", fmt.Errorf("failed to create prestate build directory: %w", err)
+		}
+
+		// Check if we already have prestate files
+		matches, _ := filepath.Glob(filepath.Join(buildDir, "prestate-proof*.json"))
+		if len(matches) == 0 {
+			// Build all prestate files directly in the target directory
+			if err := prestateBuilder.Build(buildDir); err != nil {
+				return "", fmt.Errorf("failed to build prestates: %w", err)
+			}
+		}
+
+		// Get the relative path from server.dir to buildDir for the URL
+		relPath, err := filepath.Rel(server.dir, buildDir)
+		if err != nil {
+			return "", fmt.Errorf("failed to get relative path: %w", err)
+		}
+
+		if !cfg.dryRun {
```

Comment:
Could this be inverted to return if it IS a `dryRun`, this un-indenting everything within?

Reply:
sure, at the cost of repeating ourselves a little bit.
done